ublk/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "ublk"
+version = "0.0.0"
+description = "Rust library for userspace block drivers"
+edition = "2021"
+repository = "https://github.com/germag/ublk"
+keywords = ["ublk", "libublksrv"]
+license = "MIT"
+authors = ["German Maglione <gmaglione@redhat.com>"]
+
+[dependencies]
+libc = "0.2.134"
+io-uring = { version = "0.5.6", features = ["unstable"] }
+bitflags = "2.0.0-rc.1"
+thiserror = "1.0.37"
+
+[dev-dependencies]
+structopt = "0.3.26"
```

Comment:
Do we still need this dependency?

Reply:
I missed removing this, thanks for catching it.